src/bidiMapper/utils/cdpServer.spec.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright 2021 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { connectCdp } from './cdpClient';
+import { StubServer } from './tests/stubServer.spec';
+import { assert as chai_assert } from 'chai';
+import { assert as sinon_assert } from 'sinon';
+import { Protocol } from 'devtools-protocol';
+
+describe('CdpClient tests.', async () => {
```

Comment:
```suggestion
describe('CdpClient tests.', () => {
```

Looks like this doesn't need to be async, and IIRC, mocha doesn't support async here, if tests are defined async one needs to pass some command line argument.

Reply:
Mocha supports async test cases but needs the `async function() {}` syntax instead of the `async () => {}` syntax due to how `this` is used: https://mochajs.org/#using-async-await